srv/src/main/java/my/bookshop/handlers/AdminServiceHandler.java

```diff
@@ -56,16 +56,17 @@
 	@Resource(name = AdminService_.CDS_NAME)
```

Comment:
What about the bean name defined here? 
Should we move that also to the c'tor and use `@Qualifier` instead?

Reply:
Of, I missed that one due to the unusual annotation. I'll give it a try with constructor injection.

Reply:
This works with constructor injection and @Qualifier annotation. 👍